tensorflow_probability/python/mcmc/transformed_kernel_test.py

```diff
@@ -308,6 +328,89 @@ def test_is_calibrated(self):
             FakeInnerKernel(lambda x: -x**2 / 2, False),
             tfb.Identity()).is_calibrated)
 
+  def test_transform_then_step(self):
+    new_kernel = transform_then_step(tfb.Identity())
```

Comment:
These two tests look redundant with the next two.  Delete them?  Or are they testing something nontrivial?

Reply:
Good point, I deleted them.